components/homepg.css

```diff
@@ -0,0 +1,116 @@
+.home-content {
+  margin: 1rem;
+}
+.home-img {
+  width: 40rem;
+  height: 40rem;
+}
+
+.home-intro {
+  margin: 3rem 0;
+  gap: 3rem;
+  justify-content: space-around;
+}
+
+.middle-bar {
+  padding: 1rem 0;
+  margin: 1rem;
+  gap: 2rem;
+  font-size: 1.5rem;
+  flex-wrap: wrap;
+  color: var(--btn-primary-color);
+}
+
+.middle-bar div {
+  cursor: pointer;
+}
+.expl-icon {
+  font-size: 3rem;
+}
+
+.left-side-img {
+  height: 40rem;
+  flex: 0.2;
+}
+
+.right-side-img img {
+  object-fit: cover;
+  height: 40rem;
+  width: 100%;
+}
+.bottom-img {
+  width: 100%;
+  height: 70%;
+}
+
+.img-wrap-mid {
+  width: 100%;
+}
+
+.home-descr {
+  width: 31rem;
+  gap: 1.5rem;
+}
+.home-descr h1 {
+  color: var(--hmpg-secondary-color);
+}
+
+.home-descr p {
+  font-size: 1.5rem;
+  color: var(--offline-color);
+}
+
+.bold {
```

Comment:
seems like a generic class name, you could add this in your component lib too

Reply:
Yes,I must. refactor needed in utility classes